process_environment.py

```diff
@@ -9,14 +9,14 @@
 
 DEBUG = False
 
-envIrradiance_cmd = "envIrradiance"
-envPrefilter_cmd = "envPrefilter"
-envIntegrateBRDF_cmd = "envBRDF"
-cubemap_packer_cmd = "cubemapPacker"
-panorama_packer_cmd = "panoramaPacker"
-envremap_cmd = "envremap"
-samplesGGX_cmd = "samplesGGX"
-envBackground_cmd = "envBackground"
+envIrradiance_cmd = "./envIrradiance"
```

Comment:
calling process_environment.py from the directory that contains the binary works just fine. I used that all the time.
if you use ./ when tools are installed it will not work